geomapshark/settings.py

```diff
@@ -370,5 +370,11 @@
     "DEFAULT_AUTHENTICATION_CLASSES": (
         "oauth2_provider.contrib.rest_framework.OAuth2Authentication",
         "rest_framework.authentication.SessionAuthentication",
-    )
+        # TODO : remove this once OAuth2 settings from qgis are clear
+        "rest_framework.authentication.BasicAuthentication",
```

Comment:
oAuth2 with QGIS has been tested and workd fine, thus, you can remove this line @olivierdalang